posts/deployToEcsWithGithubAction.md

```diff
@@ -0,0 +1,177 @@
+---
+authors:
+- Amine Ouali Alami
+tags:
+- Spring Boot
+- JHipster
+- GitHub
+- GitHub Actions
+- DevOps
+- AWS
+- Amazon ECS
+- AWS Fargate
+- Amazon ECR
+
+date: 2020-20-10T20:16:10.000Z
+title: "Deploy to ECS using GitHub Actions"
+image: https://raw.githubusercontent.com/ippontech/blog-usa/master/images/2020/01/deployToEcsWithGithubAction.jpg
+---
+
+Building a DevOps platform nowadays can be challenging for architecture and plateform infrastructure teams, due to the integration complexity with many platforms: source control, ci/cd, artifact repositories, quality, SAST/SCA, deployment ...  , Many DevOps leader believes that the challenge of DevOps is to reduce the number of platforms involved in one pipeline.
+That's why tech giants are trying to integrate all the pipeline steps into their platforms.
+In this article, we will deploy a simple [Spring Boot](https://spring.io/projects/spring-boot) application to [Amazon Elastic Container Service (ECS)](https://spring.io/projects/spring-boot) using the new GitHub feature called [GitHub Actions](https://github.com/features/actions).
+
+# What is GitHub Actions
+GitHub Actions enables you to create custom software development lifecycle pipelines directly in your Github repository.
+You can write individual tasks, called actions, and combine them to create a custom pipeline.
+Behind the scenes, your pipeline is run on GitHub-hosted machines, called runners.
+
+
+# Create your first pipeline with Github actions
+On your GitHub repository select the Actions tab.
+GitHub propose popular pipelines to start with. We will choose: Deploy to Amazon ECS
```

Comment:
GitHub proposes popular pipelines right out of the gate. We will choose: **Deploy to Amazon ECS**